recipes/libxml2/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "2.9.3" %}
+
+package:
+    name: libxml2
+    version: {{ version }}
+
+source:
+    fn: libxml2-{{ version }}.tar.xz
+    url: https://git.gnome.org/browse/libxml2/snapshot/libxml2-{{ version }}.tar.xz
+    md5: 59d281614c87d0c767134c55de20a8a9
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - autoconf
+        - automake
+        - libtool
+        - pkg-config
+        - python
+        - icu
+        - libiconv
+        - xz
+        - zlib
+    run:
+        - python
+        - icu
+        - libiconv
+        - xz
+        - zlib
+
+test:
+    imports:
+        - libxml2
+    files:
+        - test.xml
+    commands:
+        - xmllint test.xml
+
+about:
+    home: http://xmlsoft.org/
+    license: MIT
+    summary: The XML C parser and toolkit of Gnome
+
+extra:
+    recipe-maintainers:
+        - ocefpaf
```

Comment:
Not sure. Probably too busy pulling my hair out over strange Mac build errors. :laughing: